pkg/sdk/streamlits_validations_gen.go

```diff
@@ -0,0 +1,67 @@
+package sdk
+
+var (
+	_ validatable = new(CreateStreamlitOptions)
+	_ validatable = new(AlterStreamlitOptions)
+	_ validatable = new(DropStreamlitOptions)
+	_ validatable = new(ShowStreamlitOptions)
+	_ validatable = new(DescribeStreamlitOptions)
+)
+
+func (opts *CreateStreamlitOptions) validate() error {
+	if opts == nil {
+		return ErrNilOptions
+	}
+	var errs []error
+	if !ValidObjectIdentifier(opts.name) {
+		errs = append(errs, ErrInvalidObjectIdentifier)
+	}
+	if everyValueSet(opts.IfNotExists, opts.OrReplace) {
+		errs = append(errs, errOneOf("CreateStreamlitOptions", "IfNotExists", "OrReplace"))
+	}
+	return JoinErrors(errs...)
+}
+
+func (opts *AlterStreamlitOptions) validate() error {
+	if opts == nil {
+		return ErrNilOptions
+	}
+	var errs []error
+	if !ValidObjectIdentifier(opts.name) {
+		errs = append(errs, ErrInvalidObjectIdentifier)
+	}
+	if !exactlyOneValueSet(opts.RenameTo, opts.Set) {
+		errs = append(errs, errExactlyOneOf("AlterStreamlitOptions", "RenameTo", "Set"))
+	}
+	return JoinErrors(errs...)
+}
+
+func (opts *DropStreamlitOptions) validate() error {
+	if opts == nil {
+		return ErrNilOptions
+	}
+	var errs []error
+	if !ValidObjectIdentifier(opts.name) {
+		errs = append(errs, ErrInvalidObjectIdentifier)
+	}
+	return JoinErrors(errs...)
+}
+
+func (opts *ShowStreamlitOptions) validate() error {
+	if opts == nil {
+		return ErrNilOptions
+	}
+	var errs []error
```

Comment:
aren't there any validations from predefined objects? (Like, In, From)

Reply:
added validation for like